ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -43,3 +43,12 @@ tests:
   commands: export ARTIFACT_DIR=/tmp/artifacts; ./test-frontend.sh
   container:
     from: test-bin
+  #
+  # Try CRUD tests first
+  #
+  #  builder_run 'GUI-Tests' 1 ./test-gui.sh crud
+- artifact_dir: /tmp/artifacts
+  as: crud
+  commands: export ARTIFACT_DIR=/tmp/artifacts; ./test-gui.sh crud
+  openshift_ansible_src:
```

Comment:
replace `openshift_ansible_src` with `openshift_installer_src` to use a 4.0 cluster instead, I'm testing with that now

Reply:
@sallyom updated master yaml to openshift_installer_src and regenerated jobs yaml